<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankManagerDialog</class>
 <widget class="QDialog" name="BankManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonBankManagerDialog">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButtonBankManagerDialog{
background-color:black;
color:white;
border-radius:10px;
}
#pushButtonBankManagerDialog:hover{

background-color:rgb(85, 170, 255);
}

#pushButtonBankManagerDialog:pressed{
	background-color: rgb(0, 85, 255);

}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>781</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 0, 0);
font-weight: bold;
font-size: 16pt;
font-family: &quot;Arial&quot;, sans-serif;
text-transform: uppercase;
letter-spacing: 1px;
</string>
   </property>
   <property name="text">
    <string>MANAGER</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditManagerViewAcc">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonManagerViewAcc">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButtonManagerViewAcc{
color: rgb(0, 0, 0);
border-background: transparent;
border-radius :12px;
border: 2px solid black;
font: 700 11pt &quot;Arial&quot;;
}


#pushButtonManagerViewAcc:hover{

background-color:rgb(85, 170, 255);
}

#pushButtonManagerViewAcc:pressed{
	background-color: rgb(0, 85, 255);

}</string>
   </property>
   <property name="text">
    <string>View Admin Accounts</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>371</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget {
    border-style: solid;
    border-width: 2px;
    border-color: black
}
</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditManagerAddAdminPin">
       <property name="styleSheet">
        <string notr="true">#lineEditManagerAddAdminPin{
	background-color: transparent;
border: 2px solid black;
border-radius: 10px;

}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 700 10pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Enter Admin Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 700 10pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Enter Admin PIN :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditManagerAddAdminName">
       <property name="styleSheet">
        <string notr="true">#lineEditManagerAddAdminName{
	background-color: transparent;
border-radius: 10px ;
border: 2px solid black

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonManagerAddAcc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButtonManagerAddAcc{
color: rgb(0, 0, 0);
border-background: transparent;
border-radius :12px;
border: 2px solid black;
font: 700 11pt &quot;Arial&quot;;
}


#pushButtonManagerAddAcc:hover{

background-color:rgb(85, 170, 255);
}

#pushButtonManagerAddAcc:pressed{
	background-color: rgb(0, 85, 255);

}</string>
    </property>
    <property name="text">
     <string>ADD ACCOUNT</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>240</y>
     <width>371</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_2{
  border-style: solid;
    border-width: 2px;
    border-color: black
}</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 700 10pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Enter Admin Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditManagerRemoveAdmin">
       <property name="styleSheet">
        <string notr="true">#lineEditManagerRemoveAdmin{
	background-color: transparent;
border-radius: 10px ;
border: 2px solid black

}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonManagerRemoveAcc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButtonManagerRemoveAcc{
color: rgb(0, 0, 0);
border-background: transparent;
border-radius :12px;
border: 2px solid black;
font: 700 11pt &quot;Arial&quot;;
}


#pushButtonManagerRemoveAcc:hover{

background-color:rgb(85, 170, 255);
}

#pushButtonManagerRemoveAcc:pressed{
	background-color: rgb(0, 85, 255);

}</string>
    </property>
    <property name="text">
     <string>REMOVE ACCOUNT</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonManagerEditAcc">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>490</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButtonManagerEditAcc{
color: rgb(0, 0, 0);
border-background: transparent;
border-radius :12px;
border: 2px solid black;
font: 700 11pt &quot;Arial&quot;;
}


#pushButtonManagerEditAcc:hover{

background-color:rgb(85, 170, 255);
}

#pushButtonManagerEditAcc:pressed{
	background-color: rgb(0, 85, 255);

}</string>
   </property>
   <property name="text">
    <string>SAVE CHANGES</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditManagerViewAcc_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>151</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditManagerUsernameEdit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>240</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEditManagerUsernameEdit{
	background-color: transparent;
border-radius: 10px ;
border: 2px solid black

}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonManagerSearch">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButtonManagerSearch{
color: rgb(0, 0, 0);
border-background: transparent;
border-radius :10px;
border: 2px solid black;
font: 700 11pt &quot;Arial&quot;;
}


#pushButtonManagerSearch:hover{

background-color:rgb(85, 170, 255);
}

#pushButtonManagerSearch:pressed{
	background-color: rgb(0, 85, 255);

}</string>
   </property>
   <property name="text">
    <string>SEARCH ADMIN</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
